src/NuGet.Core/NuGet.Versioning/VersionRangeComparer.cs

```diff
@@ -90,8 +93,11 @@ public int GetHashCode(VersionRangeBase obj)
 
             combiner.AddObject(obj.IsMinInclusive);
             combiner.AddObject(obj.IsMaxInclusive);
+#pragma warning disable CS8604 // Possible null reference argument.
+            // Do we have a null bug here?
             combiner.AddObject(_versionComparer.GetHashCode(obj.MinVersion));
             combiner.AddObject(_versionComparer.GetHashCode(obj.MaxVersion));
+#pragma warning restore CS8604 // Possible null reference argument.
```

Comment:
`IEqualityComparer<T>.GetHashCode(T)` doesn't allow nulls, even in TFMs where the BCL is annotated. However, our implementation "gracefully" handles nulls. So, not sure what to do here. Should I wrap them?
```suggestion
            if (obj.MinVersion != null) { combiner.AddObject(_versionComparer.GetHashCode(obj.MinVersion)); }
            if (obj.MaxVersion != null) { combiner.AddObject(_versionComparer.GetHashCode(obj.MaxVersion)); }
```

Reply:
I'm ok with wrapping them.
We'd be doing the same thing in the rest of our code.